azure/template.json

```
@@ -244,6 +256,14 @@
                 "name": "WEBSITE_SLOT_POLL_WORKER_FOR_CHANGE_NOTIFICATION",
                 "value": "0"
               }
```

missing a comma here
```suggestion
              },
```

---

:doh: